flags.go

```diff
@@ -490,14 +518,15 @@ func populateFlags(c *cli.Context) (flags *flagStorage, err error) {
 		SequentialReadSizeMb:               int32(c.Int("sequential-read-size-mb")),
 
 		// Tuning,
-		MaxRetrySleep:              c.Duration("max-retry-sleep"),
-		StatCacheCapacity:          c.Int("stat-cache-capacity"),
-		StatCacheTTL:               c.Duration("stat-cache-ttl"),
-		TypeCacheTTL:               c.Duration("type-cache-ttl"),
-		HttpClientTimeout:          c.Duration("http-client-timeout"),
-		MaxRetryDuration:           c.Duration("max-retry-duration"),
-		RetryMultiplier:            c.Float64("retry-multiplier"),
-		LocalFileCache:             c.Bool("experimental-local-file-cache"),
+		MaxRetrySleep:     c.Duration("max-retry-sleep"),
```

Comment:
Seems formatting error.

Reply:
No, the formatting is correct. I checked by running git formatting also. This change has happened because of the line removals.